.github/workflows/ci_cd.yml

```diff
@@ -174,8 +178,7 @@ jobs:
       uses: ansys/actions/release-pypi-test@main
       with:
         library-name: ${{ env.test-library-name }}
-        twine-username: "__token__"
-        twine-token: ${{ secrets.PYANSYS_PYPI_TEST_PAT }}
+        use-trusted-publisher: true
```

Comment:
I just added trusted publishing support also for the PyPI project. Could we update those lines too?